ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py

```diff
@@ -2795,3 +2795,36 @@ def test_paddle_tensor_numel(
         frontend=frontend,
         on_device=on_device,
     )
+
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="paddle.to_tensor",
+    method_name="trunc",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
```

Comment:
Also, would be better practice to use `get_dtypes("valid"),` here. Since you have added `@with_supported_dtypes` the check will be able to figure out the valid dtypes automatically.